### PR TITLE
COD-19 dummy task

### PR DESCRIPTION
**Pull request by MugeshKP**
task 1 using for loop understand

### CODE REVIEW

File: src/mugesh/HelloWorld.java [hunk 1]
```diff
@@ -5,6 +5,9 @@ public class HelloWorld {
 	public static void main(String[] args) {
 		// TODO Auto-generated method stub
 		String a="learn pull";
+		int a = 9;
+		int b =5;
+		String b ="git push";
```
Comment (iamravisankar) [inline src/mugesh/HelloWorld.java:10]: why changes here @MugeshKP 
Please verify with @VRManojKarthik

Comment (iamravisankar) [reply src/mugesh/HelloWorld.java:10]: remove line no 10

**Review by iamravisankar — CHANGES_REQUESTED** [review]
please remove hellloworld.class file

**Comment by MugeshKP** [comment]
> please remove hellloworld.class file

i am not able to find class file @iamravisankar

**Review by iamravisankar — APPROVED** [review]
looks good

**Pull request merged**